### DISCUSSION TITLE
v2.0

### DISCUSSION THREAD

**Discussion by yukulele** [WebAudio/web-audio-api#2464] [Q&A] [answered]
What about version Web Audio Api v2.0? Is it discontinued?

https://github.com/WebAudio/web-audio-api-v2

Are there any new features planned for the Web Audio API?

**Comment by padenot** [comment, marked as answer]
V2.0 was a repository used to have a location to file issues and other ideas that would _not_ be integrated in the text that is now a W3C Recommendation.

Since this is now done, and because the W3C now has a process to continously work on specification text (via proposed additions/correction), we've decided to move back all the issues from V2.0 into this repo (all links are properly redirected, etc.), and continue the work here.

You can find all the modifications since V1 in the changelog section with links: https://webaudio.github.io/web-audio-api/#changes, and all issues at the usual location: https://github.com/WebAudio/web-audio-api/issues. In particular, new features are tagged [`new-features`](https://github.com/WebAudio/web-audio-api/issues?q=is%3Aissue+is%3Aopen+label%3Anew-feature).

While there are clearly items we're working on already, we're asking the community for input so that we can work on the things that matter, here is a survey that you can take: https://docs.google.com/forms/d/e/1FAIpQLSfpBxhRrnJ3jm8uwrNaTgW8FPKwZdfzKG7_xWLJ7TxDjxnOLA/viewform.

**Reply by yukulele** [reply]
Thanks, it's clearer!
my doubt was caused by the [readme referring to v2](https://github.com/WebAudio/web-audio-api#v20) and [v2 being marked as archived](https://github.com/WebAudio/web-audio-api-v2#note-this-repository-is-archived-and-read-only-all-active-v2-issues-are-transferred-to-the-v1-repository-please-use-the-v1-repository-for-feature-request-and-bug-report) without explaining why.
Maybe this should be removed to be more clear for newcomers.

**Reply by padenot** [reply]
Thanks. I'm going to remove any reference to v2.0 from this repo's readme. It's just an historical process artifact at this point.
